samples/csharp_dotnetcore/14.nlp-with-dispatch/Controllers/BotController.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+
+namespace Microsoft.BotBuilderSamples
+{
+    [Route("api/messages")]
```

Comment:
Oh good - this makes debugging easier. I wonder if we should put a constant for this in the Integration layer?